go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/google/uuid v1.1.2 // indirect
 	github.com/googleapis/gnostic v0.2.0 // indirect
+	github.com/gorilla/websocket v1.4.0
```

Comment:
This version of gorilla/websocket has a vulnerability in it. We should use v1.4.2 instead.

Reply:
Fixed.